### PR TITLE
Updated Dutch translation

### PR DESCRIPTION
**Pull request by Vistaus**
All updated, but I do have to say that it's quite a hassle to translate .resx files. Would it be possible to look into a different solution that makes it easier, like Transifex, Weblate or Crowdin?

### CODE REVIEW

File: NickvisionMoney.Shared/Resources/Strings.nl.resx [hunk 1]
```diff
@@ -0,0 +1,677 @@
...
+	<data name="NoAccountOpened" xml:space="preserve">
+		<value>Er is geen rekening geopend</value>
+	</data>
+
+	<data name="NoAccountDescription" xml:space="preserve">
+		<value>Open een rekening of voeg er een toe om aan de slag te gaan. Ook kunt u een rekeningbestand hierheen slepen.</value>
+	</data>
+
+	<data name="RecentAccounts" xml:space="preserve">
+		<value>Recent Accounts</value>
```
Comment (fsobolev) [inline NickvisionMoney.Shared/Resources/Strings.nl.resx:164]: Not translated

File: NickvisionMoney.Shared/Resources/Strings.nl.resx [hunk 2]
```diff
@@ -0,0 +1,677 @@
...
+	<data name="TransferMoney" xml:space="preserve">
+		<value>Geld overmaken</value>
+	</data>
+
+	<data name="TransferMoney.Tooltip" xml:space="preserve">
+		<value>Geld overmaken (Ctrl+T)</value>
+	</data>
+
+	<data name="ImportFromFile" xml:space="preserve">
+		<value>Import from File</value>
```
Comment (fsobolev) [inline NickvisionMoney.Shared/Resources/Strings.nl.resx:240]: Not translated

File: NickvisionMoney.Shared/Resources/Strings.nl.resx [hunk 3]
```diff
@@ -0,0 +1,677 @@
...
+		<value>Uploaden</value>
+	</data>
+
+	<!--TransferDialog-->
+	<data name="Transfer" xml:space="preserve">
+		<value>Overboeken</value>
+	</data>
+
+	<data name="TransferDescription" xml:space="preserve">
+		<value>TDoor geld over te maken wordt het bedrag in kwestie als uitgave op de oorspronkelijke rekening genoteerd en als inkomst op de andere.</value>
```
Comment (fsobolev) [inline NickvisionMoney.Shared/Resources/Strings.nl.resx:482]: A typo in the beginning?

**Review by fsobolev — CHANGES_REQUESTED** [review]

File: NickvisionMoney.Shared/Resources/Strings.nl.resx (hunk 3 above)
Comment (Vistaus) [reply NickvisionMoney.Shared/Resources/Strings.nl.resx]: Definitely. Thanks for pointing everything out - it's fixed now!

File: NickvisionMoney.Shared/Resources/Strings.nl.resx (hunk 1 above)
Comment (Vistaus) [reply NickvisionMoney.Shared/Resources/Strings.nl.resx]: Odd. I guess my text editor didn't save the file after my last edits…

File: NickvisionMoney.Shared/Resources/Strings.nl.resx (hunk 2 above)
Comment (Vistaus) [reply NickvisionMoney.Shared/Resources/Strings.nl.resx]: Odd. I guess my text editor didn't save the file after my last edits…

**Review by fsobolev — APPROVED** [review]

**Comment by nlogozzo** [comment]
I would like to look into Weblate which would generate the `resx` for each language in the near future. It's on the list of things to do!

But in the mean time, thanks for taking the time to update the translation :)

**Review by nlogozzo — APPROVED** [review]

**Pull request merged**